app/assets/javascripts/app/utils/auto-logout.js

```diff
@@ -1,5 +1,3 @@
 export default () => {
-  window.onbeforeunload = null;
```

Comment:
I don't think we had any browser-specific issues, as long as we are making sure that we are not assigning functions to `onbeforeunload`

If your manual testing works, I'm happy, thanks!